### ISSUE TITLE
has_snapshot_children and has_one on model fails if nil

### ISSUE THREAD

**Issue by newbenji** [westonganger/active_snapshot#10]
if you have in your has_snapshot_children
 ```
  something:          {
        record:        instance.something,
        delete_method: ->(item) { item.release! }
      },
```

and for your model 

``
has_one :something
``

if fails if instance.something = nil
for for rails has_one is 0 or 1 item. so should not fail on nil

**Comment by newbenji** [comment]
solved it this way for now

but not sure how the restore will react
`record:        instance.something ? instance.something : {},`

**Comment by westonganger** [comment]
Yeah your totally right. Nil should be an accepted value. I can add this soon or will accept a PR if your willing

**Comment by westonganger** [comment]
This is now fixed in 1aa90e6

**Comment by westonganger** [comment]
v0.2.1 is now released which contains this fix.
